recipe/install-rust.sh

```diff
@@ -5,7 +5,10 @@ set -ex
 # windows shell doesn't start here
 cd $SRC_DIR
 
-./install.sh --prefix=$PREFIX
```

Comment:
Running `install.sh` whilst `rust-std` is already installed causes files from `rust-std` to get `*.old` backups, i.e., unnecessary duplication of those files in the `rust` package. Running `install.sh` once in `build.sh` circumvents this, so we're all good now.